File: app/src/main/java/com/dot2line/vocabar/base/BaseApplication.java
package com.dot2line.vocabar.base;

import android.app.Application;

import com.crashlytics.android.Crashlytics;
import io.fabric.sdk.android.Fabric;
import io.realm.Realm;

public class BaseApplication extends Application {

  @Override
  public void onCreate() {
    super.onCreate();
    Fabric.with(this, new Crashlytics());

    Realm.init(this);
  }
}

File: app/src/main/java/com/dot2line/vocabar/base/BaseApplication.kt
package com.dot2line.vocabar.base

import android.app.Application

import com.crashlytics.android.Crashlytics
import io.fabric.sdk.android.Fabric
import io.realm.Realm

class BaseApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Fabric.with(this, Crashlytics())
        Realm.init(this)
    }
}
